### DISCUSSION TITLE
Password JKBMS Bluetooth

### DISCUSSION THREAD

**Discussion by testarz** [Louisvdw/dbus-serialbattery#1004] [Q&A]
Thankfully i can connect my Raspberry Pi to my JKBMS over Bluetooth with the dbus-serialbattery. This unfortunately only works, when the password is set to 1234 in the JK Software. Is there a possibility to change the password?

Thanks for suggestions

**Comment by mr-manuel** [comment]
Just read the notes after you install the driver...

https://github.com/Louisvdw/dbus-serialbattery/blob/ac892a108ee8623effeac23ff64c899d0de32c18/etc/dbus-serialbattery/reinstall-local.sh#L523-L535

**Comment by testarz** [comment]
Thank you for the suggestion, i tried it first with my changed code and it didn´t work. then i found exatly these notes and changed the code siply back to 1234. After that, the connection worked immideatly. but when i change it now to something else again, the pi won´t connect anymore an the driver starts again and again

**Comment by chase1124** [comment]
Does someone know where/how the default pin is set in the install setup? Tried some simple grep'ing but didn't find a 1234 reference. I could not get my jkbms to connect via the debian directions linked in the install notes (https://wiki.debian.org/BluetoothUser#Pairing_using_CLI) but it works just fine if I edit the config for dbus-serialbattery and do the re-install after changing to the default '1234' pin. I know it's a bit of a dirty approach to the issue but could also be nice if there was a way  to set the new PIN somewhere in code/config as the install script already sets up bluetooth cleanly.
